packages/backend/src/routers/projectRouter.ts

```diff
@@ -91,7 +91,7 @@ projectRouter.get('/explores', isAuthenticated, async (req, res, next) => {
             await projectService.getAllExploresSummary(
                 req.user!,
                 req.params.projectUuid,
-                req.query.filtered === 'true',
```

Comment:
This is going to break the project table configuration page (ProjectTablesConfiguration) where it expects to get all the tables.